crates/re_ui/src/list_item.rs

```diff
@@ -220,9 +220,14 @@ impl<'a> ListItem<'a> {
                 text_rect.max.x -= button_response.rect.width() + ReUi::text_to_icon_padding();
             }
 
-            let text =
+            let mut text_job =
                 self.text
-                    .into_galley(ui, Some(false), text_rect.width(), egui::TextStyle::Button);
+                    .into_text_job(ui.style(), egui::FontSelection::Default, Align::LEFT);
+            text_job.job.wrap.max_width = text_rect.width();
+            text_job.job.wrap.max_rows = 1;
+            text_job.job.wrap.break_anywhere = true;
```

Comment:
shorter and more readable:

```suggestion
            text_job.job.wrap = egui::text::TextWrapping::elide_at_width(text_rect.width());
```